wikibase/1.30/bundle/download-extension.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Looking forward, this should either be download-oauth-extension. or download-extensions,? As we will either add more extensions to this file or add more files?

Reply:
This is a generic script, it should be possible to call it again with a different parameter to a different single extension

Reply:
Ahh yes, so currently it still has OAuth hard coded:

| jq -r '.query.extdistbranches.extensions.OAuth.REL1_30